frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -42,6 +55,21 @@ function EditableInstructorsList(props) {
         );
     }
 
+    function DeleteButtonCell({ original: instructorInfo }) {
+        const { id } = instructorInfo;
+        const disabled = workingInstructors.has(id);
+
+        function onClick() {
+            deleteInstructor({ id });
+        }
+
+        return (
+            <Button variant="danger" onClick={onClick} disabled={disabled}>
+                <span>&times;</span>
```

Comment:
Use the `Fa`icon for this.

Reply:
can you point me to an example of this?
global project search of "Fa" might take a while 😋